Some small updates to improve speed and readability

wroersma opened the pull request:
Let me know if you would like me to change anything. I would like to fully convert the print statements to logging outputs which gives you a lot more room for use.

Commit:
use ujson when you can to load json data faster
remove duplicate code
remove unused imports
update print statements to support python 3 print statement type

wroersma committed:
Update FSEParser_V3.1.py

gc-nibrahim commented on FSEParser_V3.1.py:
```diff
@@ -28,14 +28,12 @@
-import time
 import json
-import csv, codecs, cStringIO
+import ujson
```
Fails on import if the module does not exist. A try, except would be useful here so that ujson or json can be used.

wroersma replied:
awesome thanks I will put that in!

Commit:
try to see if ujson package is installed otherwise use json

wroersma commented:
I'm not sure the best way you want the try statement so let me know if I need to change it again

gc-nibrahim commented on FSEParser_V3.1.py:
```diff
@@ -74,216 +72,229 @@
-        # Test arguments passed #
-        if opts.report_queries==True:
-            options.error("Unable to proceed. report_queries.json was not specified using the -q option. \
+    return options
```
# Test if Python version in use is 3 or greater
if (sys.version_info > (3, 0)):
    print 'Error: Unable to run script. FSEventsParser does not currently support Python 3.x syntax.'
    print 'Python 2.7 is recommended.'
    sys.exit(0)

gc-nibrahim commented on FSEParser_V3.1.py:
```diff
@@ -74,216 +72,229 @@
-print '\n=========================================================================='
-print 'FSEParser v', VERSION, ' -- provided by G-C Partners, LLC'
-print '==========================================================================\n'
+print('\n==========================================================================' '\n'
```
Adding the parenthesis to the print statements result in the parenthesis being printed along with the str. Instead of using the parenthesis to try and conform to Python 3 standards, they may need to be removed. Even with the parenthesis in the print statements, running the script using python 3 still fails. This applies to all print states in the script. Refer to comment on line 128/129 for suggestion on testing if python 3 is in use.

gc-nibrahim committed:
Minor updates

Reverting some of the changes from last pull request.
-----------
- Updated print statements. Now the parenthesis are not printed.
- restored print statements and logfile.write strings to include \n for formatting purposes.
- restored output report rows to include newline as all reports had all data stored on the first column of the report

gc-nibrahim commented:
Closing. Updated script to 3.2. Addresses readability  and print statements highlighted in this pull. ujson was not added in the new release. Feel free to pull again to add ujson support if you like.
